### PR TITLE
Added closePaint script and removed no longer used spray paint scripts

### PR DESCRIPTION
**Pull request by ericrius1**
closePaint.js allows a user running this script to stand close to entities and paint onto them with the hydras. It is a first pass at what will become a more fully featured painting-on-surfaces app in the future.

### CODE REVIEW

**Comment by ericrius1** [comment]
Test using this url: https://raw.githubusercontent.com/ericrius1/hifi/sprayPaint/examples/closePaint.js

File: examples/closePaint.js [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+
+var MIN_STROKE_WIDTH = 0.005;
+var MAX_STROKE_WIDTH = 0.03;
+
+
+var center = Vec3.sum(MyAvatar.position, Vec3.multiply(2, Quat.getFront(Camera.getOrientation())));
+
+
+
+function MyController(hand, triggerAction) {
```
Comment (ZappoMan) [inline examples/closePaint.js]: the formatting of this file is wrong... it needs to be updated to 4 space indents

File: examples/closePaint.js [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+    s: 0.8,
+    l: 0.4
+  };
+
+
+  this.laserPointer = Overlays.addOverlay("circle3d", {
+    color: hslToRgb(this.strokeColor),
+    solid: true,
+    position: center
+  })
```
Comment (ZappoMan) [inline examples/closePaint.js]: coding standard: statements end with ;

File: examples/closePaint.js [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+  this.triggerValue = 0;
+  this.prevTriggerValue = 0;
+  var _this = this;
+
+  this.update = function() {
+    this.updateControllerState()
```
Comment (ZappoMan) [inline examples/closePaint.js]: coding standard: statements end with ;

File: examples/closePaint.js [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+    Entities.editEntity(this.currentStroke, {
+      linePoints: this.strokePoints,
+      normals: this.strokeNormals,
+      strokeWidths: this.strokeWidths
+    });
+    if (this.strokePoints.length === MAX_POINTS_PER_LINE) {
+      this.painting = false;
+      return;
+    }
+    this.oldPosition = position
```
Comment (ZappoMan) [inline examples/closePaint.js]: coding standard: statements end with ;

File: examples/closePaint.js [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+    this.strokes.push(this.currentStroke);
+
+  }
+
+  this.updateControllerState = function() {
+    this.triggerValue = Controller.getActionValue(this.triggerAction);
+    if (this.triggerValue > TRIGGER_ON_VALUE && this.prevTriggerValue <= TRIGGER_ON_VALUE) {
+      this.squeeze();
+    } else if (this.triggerValue < TRIGGER_ON_VALUE && this.prevTriggerValue >= TRIGGER_ON_VALUE) {
+      this.release()
```
Comment (ZappoMan) [inline examples/closePaint.js]: coding standard: statements end with ;

File: examples/closePaint.js [hunk 6]
```diff
@@ -0,0 +1,353 @@
...
+ * adapted from http://en.wikipedia.org/wiki/HSL_color_space.
+ * Assumes h, s, and l are contained in the set [0, 1] and
+ * returns r, g, and b in the set [0, 255].
+ *
+ * @param   Number  h       The hue
+ * @param   Number  s       The saturation
+ * @param   Number  l       The lightness
+ * @return  Array           The RGB representation
+ */
+function hslToRgb(hsl, hueOffset) {
```
Comment (ZappoMan) [inline examples/closePaint.js]: move this to utilities or utils

**Pull request merged**